src/database/DataTypes/Unit.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import Realm from 'realm';
+
+/**
+ * TODO
+ */
+export class Unit extends Realm.Object {}
+
+Unit.schema = {
+  name: 'Unit',
+  primaryKey: 'id',
+  properties: {
+    id: 'string',
+    units: { type: 'string', default: 'Each' },
```

Comment:
I'm wondering if this should actually be optional rather than defaulted. In mSupply desktop there doesn't seem to be a default "Each" set anywhere? As far as I can tell you cannot make/save a unit with this field blank.

Reply:
oops I forgot about this - I didn't know what to do so I did that!

Reply:
Done in commits:

- 257ec37fe8a3161ce10fa8ed0b51f8e67f6c3df3
- 14de00471670b039d0c2a74e6a0793bc3f83eb51
- c0d00463540ac713ca67477db8d206f144ab896a